src/app/zap-templates/zcl/data-model/chip/fan-control-cluster.xml

```diff
@@ -44,8 +44,8 @@ limitations under the License.
     <item name="Off/Low/High" value="0x01"/>
     <item name="Off/Low/Med/High/Auto" value="0x02"/>
     <item name="Off/Low/High/Auto" value="0x03"/>
-    <item name="Off/On/Auto" value="0x04"/>
-    <item name="Off/On" value="0x05"/>
+    <item name="Off/High/Auto" value="0x04"/>
```

Comment:
The XML change should probably be in a separate PR from the implementation change, so it can be carefully reviewed in terms of its impact on SDK APIs.

Reply:
Noted, will revert for this PR.